content/en/blog/2025/kubeletstats-receiver-metrics-deprecation.md

```diff
@@ -0,0 +1,146 @@
+---
+title:
+  'Kubernetes CPU Metrics in the kubeletstats Receiver: Transition from
+  .cpu.utilization to .cpu.usage'
+linkTitle: kubeletstats Receiver metrics update
+author: >
+  [Christos Markou](https://github.com/ChrsMark) (Elastic), [Tyler
+  Helmuth](https://github.com/TylerHelmuth) (Honeycomb), [Dmitrii
+  Anoshin](https://github.com/dmitryax) (Cisco/Splunk)
+date: 2025-05-30
+issue: https://github.com/open-telemetry/opentelemetry.io/issues/6847
+cSpell:ignore: Anoshin Dmitrii Helmuth Kubelet kubeletstats Kubelet’s Markou
+---
+
+The OpenTelemetry Collector’s
+[`kubeletstats`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.127.0/receiver/kubeletstatsreceiver/README.md)
```

Comment:
Is there a reason why the URL here refers to v0.127.0, but v0.126.0 is used elsewhere in the page?

Reply:
Btw, thanks for pinning those URLs to a specific version. That will help the links stay valid as the blog post ages. 🙌🏻

Reply:
I think at the time of writing the latest version was `v0.126.0` and after few days (this week) `v0.127.0` came out, so that's why we ended up with 2 versions most probably :). Thank's for spotting this!

Reply:
Right. That should be fixed now.